README.md

```diff
@@ -1,131 +1,236 @@
 <div align="center">
 	<img src="https://github.com/frappe/design/raw/master/logos/png/bench-logo.png" height="128">
-	<h2>bench</h2>
+	<h2>Bench</h2>
 </div>
 
-bench is a command-line utility that helps you to install apps, manage multiple sites and update Frappe / ERPNext apps on */nix (macOS, Ubuntu, Debian, CentOS, etc) for development and production.
+Bench is a command-line utility that helps you to install, update, and manage multiple sites for Frappe/ERPNext applications on [*nix systems](https://en.wikipedia.org/wiki/Unix-like) for development and production.
```

Comment:
```suggestion 
bench is a command-line utility that helps you to install, update, and manage multiple sites for Frappe/ERPNext applications on [*nix systems](https://en.wikipedia.org/wiki/Unix-like) for development and production.
```

Reply:
i really dont think it should be lowercase. its a command line utility, i get that, but i dont see the point in capitalization.

Reply:
The reason for de-capitalization was that it's a command-line utility. It's 'Bench' if you believe that's what it should be.